src/extensions/cellpack/model.ts

```diff
@@ -108,7 +109,7 @@ async function getStructure(plugin: PluginContext, model: Model, source: Ingredi
     }
     let query;
     if (source.selection){
-        const asymIds: string[] = source.selection.replace(' :', '').split(' or');
+        const asymIds: string[] = source.selection.replace(' ', '').replace(':', '').split('or');
```

Comment:
How about changing, in the files, the `selection` property to `authAsymIds` with type `string[]`? Or do you need it compatible with something else?

Reply:
Definitly, I will have to make sure first that all the pipeline will work without issues. How this will look like in json ? like this : {"selection":["A","B",...]} ?